parley/src/context.rs

```diff
@@ -135,7 +144,8 @@ impl<'a, B: Brush, T: TextSource> RangedBuilder<'a, B, T> {
                 char_index += 1;
             }
         }
-        use super::layout::{Decoration, Style};
+
+        // Define a function that converts `ResolvedDecoration` into `Decoration` (used just below)
```

Comment:
I'm not sure how useful this comment is - maybe it would be better to rename the function?